docs/src/main/paradox/keyspaces.md

```diff
@@ -0,0 +1,34 @@
+# Amazon Keyspaces
+
+@@@ warning
+
+**This project is not continuously tested against Amazon Keyspaces.**
+
+@@@
+
+[Amazon Keyspaces](https://aws.amazon.com/keyspaces/) is a wire compatible replacement for Cassandra.
+
+Initial testing shows that most things are working. To setup authentication follow the [Developer Guide](https://docs.aws.amazon.com/keyspaces/latest/devguide/using_java_driver.html).
+
+When using Amazon Keyspaces you need to additionally configure:
+
+```
+datastax-java-driver {
+  # QUORUM is not supported: https://docs.aws.amazon.com/keyspaces/latest/devguide/consistency.html
+  profiles.akka-persistence-cassandra-profile.basic.request.consistency = LOCAL_QUORUM
+
+  advanced {
+    metadata {
+      # not supported: https://forums.aws.amazon.com/thread.jspa?messageID=945795
+      schema.enabled = false
+      token-map.enabled = false
+    }
+    
+    # https://discuss.lightbend.com/t/akka-cluster-using-cassandra-driver-and-running-on-eks-with-keyspaces-for-cassandra-generates-weird-warning/7182
+    connection.warn-on-init-error = false
+  }
+}
+
+# in case you use akka management health check (https://doc.akka.io/docs/akka-persistence-cassandra/current/healthcheck.html) you need to change the query as `now()` is not supported: https://forums.aws.amazon.com/thread.jspa?threadID=329369&tstart=0
+akka.persistence.cassandra.healthcheck.health-check-cql = "SELECT * FROM system.local"
```

Comment:
In case you know a more lightweight query feel free to edit. None of those worked for me https://docs.aws.amazon.com/keyspaces/latest/devguide/cql.functions.html ...